src/styles/StyledSquare.js

```diff
@@ -1,7 +1,34 @@
 import styled from "styled-components";
+import coin from "../cards/images/coin.jpg";
+import tree from "../cards/images/tree.png";
+import x from "../cards/images/x.jpg";
 
-export const StyledSquare = styled.div`
-  flex: 1 0 25%;
+export const StyledSquare = styled.button`
   text-align: center;
-  margin: 6px 0;
+  margin: 0;
+  padding: 0;
+  height: 30px;
+  width: 30px;
+  background-image: ${(props) => {
+    const type = props.type;
+    if (type === 2) {
+      return `url(${coin})`;
+    } else if (type === 3) {
+      return `url(${tree})`;
+    } else if (type === 4) {
+      return `url(${x})`;
+    }
+  }};
+  background-size: 90%;
```

Comment:
Test this out with 70% and let me know what you think

Reply:
Love it!

Reply:
Neat.

```suggestion
  background-size: 70%;
```